pom.xml

```diff
@@ -156,6 +156,7 @@
                 <version>2.22.0</version>
                 <configuration>
                     <skipTests>${skipUnitTests}</skipTests>
+                    <trimStackTrace>false</trimStackTrace>
```

Comment:
This will be deleted afterwards.

Reply:
before you push this pr or in a separate PR?

Reply:
I'll deleted this before pushing this PR.